test/index.test.ts

```diff
@@ -1,5 +1,6 @@
-import lunh from '../index';
 import { successfulStripeCreditCard, failedStripeCreditCard } from './stripe';
+import lunh = require('../index');
```

Comment:
Is there no actual other way to not have this ? It's so unconventinal I feel like we are giving up convention for ES6 module just to make node module works